cfme/configure/settings.py

```diff
@@ -62,26 +62,29 @@ def __init__(self, description=None, scope=None, days=None, hours=None, timezone
         self.hours = hours
         self.timezone = timezone
 
-    def create(self):
+    def create(self, negative=False):
```

Comment:
I think in context like this I've seen the word/variable `cancel` used, where the form/action is not saved/confirmed when cancel is true.